<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtConverterV2Class</class>
 <widget class="QMainWindow" name="QtConverterV2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtConverterV2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="l_bin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Binary</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_hex">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Hex</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decimal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="binary">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="hex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="l_dec">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Decimal</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="octal">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="l_octal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Octal</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rb_dec">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_bin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_hex">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_octal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Binary Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Bites:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>4</number>
     </property>
     <property name="maximum">
      <number>32</number>
     </property>
     <property name="value">
      <number>8</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>449</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtConverterV2.qrc"/>
 </resources>
 <connections/>
</ui>
